saleor/graphql/webhook/sorters.py

```diff
@@ -0,0 +1,20 @@
+import graphene
+
+from ..core.types import SortInputObjectType
+
+
+class EventDeliverySortField(graphene.Enum):
+    CREATED_AT = ["created_at"]
+
+    @property
+    def description(self):
+        if self.name in EventDeliverySortField.__enum__._member_names_:
+            sort_name = self.name.lower().replace("_", " ")
+            return f"Sort pages by {sort_name}."
```

Comment:
We're sorting event deliveries not pages :)

Reply:
fixed